cmake/cpp.cmake

```diff
@@ -172,6 +186,9 @@ foreach(FLAVOR ${FLAVORS})
     target_link_libraries(${FLAVOR} PUBLIC -liphlpapi -lrpcrt4)
   endif()
 
+  #Nlopt include 
+  target_include_directories(${FLAVOR} PRIVATE ${NLopt_INCLUDE_DIRS})
+
```

Comment:
Due to the `NLopt::nlopt` in `target_link_libraries` that should carry the dependency include directories, this `target_include_directories` should be useless and safely removed I think.

Reply:
Exact. I've done the modification. Thank for your comment.